app/src/main/java/com/github/se/polyfit/model/nutritionalInformation/NutritionalInformation.kt

```diff
@@ -69,8 +69,17 @@ class NutritionalInformation {
     return newNutritionalInformation
   }
 
+  operator fun minus(other: NutritionalInformation): NutritionalInformation {
+    val newNutritionalInformation = NutritionalInformation(mutableListOf())
+
+    nutrients.forEach { newNutritionalInformation.update(it) }
+    other.nutrients.forEach { newNutritionalInformation.update(it * -1.0) }
+
+    return newNutritionalInformation
+  }
```

Comment:
Not much handling, I figured that in theory if we are only removing ingredients that are in the Nutrition facts it should be fine, but I agree that should have a case for that. I'll leave a ticket for backlog since I think it shouldn't block this for now

Reply:
https://github.com/orgs/swent-group10/projects/2?pane=issue&itemId=59479567